src/tapis-ui/_common/Breadcrumbs/Breadcrumbs.module.scss

```diff
@@ -0,0 +1,5 @@
+.root {
+  text-transform: none;
+  font-size: 0.8em;
+  line-height: 1.8em;
+}
```

Comment:
Let's not define a style here. the parent component should do that.

(Except maybe the text-transform: none)